addons/hr/models/hr.py

```diff
@@ -110,8 +110,9 @@ def _default_image(self):
 
     # resource and user
     # required on the resource, make sure required="True" set in the view
-    name = fields.Char(related='resource_id.name', store=True, oldname='name_related', readonly=False)
+    name = fields.Char(related='resource_id.name', store=True, oldname='name_related', readonly=False, tracking=True)
     user_id = fields.Many2one('res.users', 'User', related='resource_id.user_id', store=True, readonly=False)
+    user_partner_id = fields.Many2one(related='user_id.partner_id', related_sudo=False, string="User's partner")
     active = fields.Boolean('Active', related='resource_id.active', default=True, store=True, readonly=False)
     # private partner
     address_home_id = fields.Many2one(
```

Comment:
Shouldn't we track the private address ?

Reply:
We decided that the private address is not self editable (at least for now)

Reply:
And if a HR officer modifies the private address, we could wan't to track it nope ? 

What do you think ?

Reply:
Good point. I'll enable tracking for the private address ;)